tests/Handler/MultipleOutputTestHandler.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tomaj\NetteApi\Test\Handler;
+
+use Tomaj\NetteApi\Handlers\BaseHandler;
+use Tomaj\NetteApi\Output\JsonOutput;
+use Tomaj\NetteApi\Response\JsonApiResponse;
+use Tomaj\NetteApi\Response\ResponseInterface;
+
+class MultipleOutputTestHandler extends BaseHandler
+{
+    public function summary(): string
+    {
+        return 'Multiple output test handler';
+    }
+
+    public function description(): string
+    {
+        return 'This API handler is for test multiple ';
+    }
+
+    public function handle(array $params): ResponseInterface
+    {
+        return new JsonApiResponse(200, ['hello' => 'world']);
+    }
+
+    public function outputs(): array
+    {
+        return [
+            new JsonOutput(200, '{"type": "object"}'),
+            new JsonOutput(200, '{"type": "string"}'),
+        ];
```

Comment:
What if I'll add 200 RedirectOutput? I know it is not correct but it is possible. Maybe we should collect all outputs and group then by code and then if there are more than one output for code, we should use oneOf key in schema